packages/common/src/index.ts

```diff
@@ -1124,29 +1124,32 @@ function formatRawValue(
         // We want to return the datetime in UTC to avoid timezone issues in the frontend like in chart tooltips
         return dayjs(value).utc(true).format();
     }
+
     return value;
 }
 
-export function formatRows(
-    rows: { [col: string]: any }[],
-    itemsMap: ItemsMap,
-): ResultRow[] {
-    return rows.map((row) =>
-        Object.keys(row).reduce<ResultRow>((acc, columnName) => {
-            const col = row[columnName];
-
-            const item = itemsMap[columnName];
-            return {
-                ...acc,
-                [columnName]: {
-                    value: {
-                        raw: formatRawValue(item, col),
-                        formatted: formatItemValue(item, col),
-                    },
+export function formatRows(rows: { [col: string]: any }[], itemsMap: ItemsMap) {
```

Comment:
this method was recently used here https://github.com/lightdash/lightdash/blob/main/packages/backend/src/services/CsvService/CsvService.ts#L476 to format rows before donwloading pivot CSV files, in case you want to test that too.

Reply:
Yep, thank you for that, will give it a go! hopefully it should be fine as the return type didn't change but always good to test, will also trigger all e2e

Reply:
Just tested the pivoted table csv export
![image](https://github.com/user-attachments/assets/e4c184cb-d01d-45e8-95a6-af6dbaaa9432)
[pivoted_csv.csv](https://github.com/user-attachments/files/18209404/pivoted_csv.csv)

Seems good 🙌